almalinux/Dockerfile

```diff
@@ -7,7 +7,8 @@
 # under the terms of the MIT License; see LICENSE file for more details.
```

Comment:
Why this has been moved? Do we plan to change the current structure of the repository? How/where would we put e.g. an AlmaLinux image targeting python3.8?

Reply:
Well, I did this after the chat we had: the idea is to provide one image, versioned, that works. For example `inveniosoftware/almalinux:v1.0.0`: such image can contain, for example, AlmaLinux 9, Python 3.9, NodeJS 18.
Upgrading it to Python 3.10 would then mean that the tag will become `v2.0.0`.
@mfenner is my interpretation of our chat correct here?

It looks like that we don't want to create, maintain and publish a matrix of images.

Reply:
Yes. We didn‘t discuss the version numbering, e.g. whether Python 3.10 would mean v2.0.0 or v1.1.0, but that is an implementation detail. Would follow the SemVer best practices, so minor image version bumps for version updates of dependencies, and major version bumps for breaking changes.

Reply:
Yes of course, this is what I meant, mine was just an example. @avivace will this work for you?